### ISSUE TITLE
Bug Report: Daddy Dearest has an embarrassing voice crack in DadBattle Erect

### ISSUE THREAD

**Issue by Hundrec** [FunkinCrew/Funkin#3757]
### Issue Checklist

- [X] I have properly named my issue
- [X] I have checked the Issues/Discussions pages to see if my issue has already been reported

### Platform

Itch.io (Downloadable Build) - Windows

### Browser

None

### Version

0.5.3

### Description (include any images, videos, errors, or crash logs)

Just stumbled upon a strange blip in DadBattle Erect, and it turns out Daddy Dearest's Voices track has a tiny blip at Section 94!
Utter shock or funny mistake?

https://github.com/user-attachments/assets/7c5ceb59-8c9e-4658-926c-d8f8168c4db6

Also found another blip at Section 78 where he sings the same pitch at a different octave:

https://github.com/user-attachments/assets/5fb71619-792c-44a0-b171-b0a2319c88b1

If these are actually meant to convey Dad being shocked, I think he should sing a note there to make them more noticeable.

### Related Detail

In Sections 11 and 59, Dad's Voices track has some strange background noise. This was also present in the original version of DadBattle at Sections 19 and 51. In fact, faint hints of the instrumental can be heard in both Voices tracks of DadBattle, so I'd say it's part of Funkin's identity at this point.
Just wanted to let you know!

https://github.com/user-attachments/assets/960af57e-6565-4729-a967-ead2a9ad67ba

### Context

These details are also present in the [2021 upload](https://www.youtube.com/watch?v=L3mmBIADJ-A) of the song (at 1:57, for example)

### Steps to Reproduce

1. Open DadBattle Erect in the Chart Editor
2. Go to Section 94
3. Disable all tracks except the Enemy's

**Comment by LukerUpgradez** [comment]
Somebody needs to get Daddy Dearest some water. Performers have to protect their throats!

**Comment by Lenancha** [comment]
So all those FLP recreations of this song are all wrong because they don't have that little blip. Damn I guess they're all unusable now.

**Comment by rockybalboba** [comment]
I'm not sure if it was intentional but I've never even noticed when playing the song
a possibility could be that dd's samples are in a slicer and the markers are in a way where that tiny bit of the next sample (B) bleeds into the previous one's (A) so if you make a sustain note on the piano roll that's longer than A that tiny bit of B plays at the end

**Comment by Hundrec** [comment]
Yeah, I'm confident it's a piano roll technical issue. Pretty funny!

**Comment by NoobyProducoesOFC** [comment]
why is that funny lol

**Comment by Hundrec** [comment]
Closing this because it's too funny
